MekHQ/src/mekhq/gui/sorter/FactionSorter.java

```diff
@@ -0,0 +1,21 @@
+package mekhq.gui.sorter;
+
+import java.util.Comparator;
+
+import mekhq.campaign.Campaign;
+import mekhq.campaign.universe.Faction;
+
+public class FactionSorter implements Comparator<Faction> {
```

Comment:
Let's put some kind of comments on this class - the sort criterion is not immediately clear.